Print_A_Bot/controls/views.py

```diff
@@ -31,3 +42,43 @@ def post(self, request):
 		pass
 
 
+class NewLightShowView(View):
+    def get(self, request):
+        context = {
+            'form': LightShowForm,
+        }
+        return render(request, 'create_lightshow.html', context)
+
+    def post(self, request):
+        form = LightShowForm(request.POST)
+
+        # revert back to blank form if not valid
+        if not form.is_valid():
+            context = {
+                'form': LightShowForm,
```

Comment:
this actually wipes any existing error, you can pass `form` back and it will show what was wrong with previous entries